enterprise_access/settings/base.py

```diff
@@ -84,6 +84,7 @@ def root(*path_fragments):
     # Resets RequestCache utility for added safety.
     'edx_django_utils.cache.middleware.RequestCacheMiddleware',
     # Enables monitoring utility for writing custom metrics.
+    'edx_django_utils.monitoring.DeploymentMonitoringMiddleware',
     'edx_django_utils.monitoring.middleware.MonitoringCustomMetricsMiddleware',
```

Comment:
nit: Keeping comment next to relevant import
```suggestion
    'edx_django_utils.monitoring.DeploymentMonitoringMiddleware',
     # Enables monitoring utility for writing custom metrics.
    'edx_django_utils.monitoring.middleware.MonitoringCustomMetricsMiddleware',
```